### ISSUE TITLE
USER ISSUE: Volume loss

### ISSUE THREAD

**Issue by MKKanzi** [StrangeLoopGames/EcoIssues#20141]
\
**Version:** 0.9.1.9 beta release-136

**Steps to Reproduce:**
Join any server, volume works fine in menu and other applications. Also mutes other applications when server has loaded.

**Expected behavior:**
Volume, noise

**Actual behavior:**
Silence, muted

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods installed

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
@MKKanzi Thank you for the report.
1. Can you provide log files? Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
2. Please provide your PC specification (include Operating System and its version).

**Comment by MKKanzi** [comment]
Hello, I believe I resolved the issue actually. It turns out it is a
Windows 10 problem, where Windows is unable to use both the audio and mic
of a Bluetooth headset at the same time. It is a known problem and simply a
function of Windows 10. Why this is a thing, I have no idea. But upon
activating anything that used the microphone, like entering the game
server, all audio would be muted (but you'd still be able to hear me in
game if I spoke). After some troubleshooting I discovered the same thing
happened if I tried to use the Windows voice recorder. However I could go
into the "telephony" mode (on Win 10) and there would be audio in-game, but
it sounded like someone at the bottom of the well on a phone from 1982. So
in case you hear of this issue again, there you go. Thank you for your time
